src/main/java/io/jenkins/plugins/google/analyze/code/security/CodeScanBuildStep.java

```diff
@@ -41,10 +41,7 @@
 import io.jenkins.plugins.google.analyze.code.security.model.ConfigAggregator;
 import io.jenkins.plugins.google.analyze.code.security.model.FileInfo;
 import io.jenkins.plugins.google.analyze.code.security.model.IACValidationService.ValidateIACParams;
-import io.jenkins.plugins.google.analyze.code.security.model.IACValidationService.response.ErrorReportRequest;
-import io.jenkins.plugins.google.analyze.code.security.model.IACValidationService.response.IACScanReportRequest;
-import io.jenkins.plugins.google.analyze.code.security.model.IACValidationService.response.Severity;
-import io.jenkins.plugins.google.analyze.code.security.model.IACValidationService.response.Violation;
+import io.jenkins.plugins.google.analyze.code.security.model.IACValidationService.response.*;
```

Comment:
It is always better to have specific import rather than using * to include everything.